src/Avataaars/Random.elm

```diff
@@ -0,0 +1,304 @@
+module Avataaars.Random exposing (generate)
+
+{-|
+
+
+# Random
+
+@docs generate
+
+-}
+
+import Avataaars exposing (Avataaar)
+import Avataaars.Accessory exposing (Accessory(..))
+import Avataaars.Clothes exposing (Clothes(..))
+import Avataaars.Eyebrow exposing (Eyebrow(..))
+import Avataaars.Eyes exposing (Eyes(..))
+import Avataaars.Face exposing (Face)
+import Avataaars.FacialHair exposing (FacialHair(..))
+import Avataaars.Graphics exposing (Graphics(..))
+import Avataaars.HairColor exposing (HairColor)
+import Avataaars.HatColor exposing (HatColor)
+import Avataaars.Mouth exposing (Mouth(..))
+import Avataaars.SkinTone exposing (SkinTone)
+import Avataaars.Top
+    exposing
+        ( Top(..)
+        , TopAccessoryFacialHair(..)
+        , TopHairColorAccessoryFacialHair(..)
+        , TopHatColorAccessory(..)
+        , TopHatColorAccessoryFacialHair(..)
+        )
+import Random exposing (Generator)
+import Random.Extra
+
+
+{-| Generates a random Avataar utilising a uniform distribution per part, per generation.
+-}
+generate : Generator Avataaar
+generate =
+    Random.map4 Avataaar Random.Extra.bool generateSkinTone generateClothes generateFace
+        |> Random.Extra.andMap generateTop
```

Comment:
You can use `Random.Extra.map5` here

Reply:
There's only a map6 or map8 function actually AFAICS... I found that weird also 😐

Reply:
I think @miniBill meant [Random.map5](https://package.elm-lang.org/packages/elm/random/latest/Random#map5). And if it needs to expand beyond 5 then switching to `Random.Extra.map6` and then `Random.Extra.andMap` beyond that.